src/main/java/graphql/introspection/Introspection.java

```diff
@@ -497,38 +542,25 @@ public enum DirectiveLocation {
                             .name("includeDeprecated")
                             .type(GraphQLBoolean)
                             .defaultValueProgrammatic(false)))
-            .field(newFieldDefinition()
-                    .name("onOperation")
-                    .type(GraphQLBoolean)
-                    .deprecate("Use `locations`."))
-            .field(newFieldDefinition()
-                    .name("onFragment")
-                    .type(GraphQLBoolean)
-                    .deprecate("Use `locations`."))
-            .field(newFieldDefinition()
-                    .name("onField")
-                    .type(GraphQLBoolean)
-                    .deprecate("Use `locations`."))
```

Comment:
`onOperation` / `onFragment` and `onField` was removed from the spec in 2016 - this has just sat around.  I think it was broken.  We have NO data fetcher defined AND there is not `getOnFragment` method that would give a value so it would always be null

I took the opportunity to remove this very old code